core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/Path2d.java

```diff
@@ -715,6 +715,7 @@ public boolean remove(double x, double y) {
 			switch (this.types[j]) {
 			case MOVE_TO:
 				this.isMultipart = null;
+				break;
 				//$FALL-THROUGH$
```

Comment:
The algorithm must be fall-through from `MOVE_TO` to the next case for working as expected. The comment `$FALL-THROUGH$` is here for enforcing this fact.
I suspect that the Travais-CI is failing due to an infinite loop you have introduced.
For putting this `break`, you need to copy paste the code of the following case. But it's break the DRY approach.

@ngaud, what is your opinion?